### ISSUE TITLE
README

### ISSUE THREAD

**Issue by kennethlove** [benoitc/offset#6]
README should actually describe what the project is.

**Comment by benoitc** [comment]
indeed. This is planned sometimes this week with the coming release :)

**Comment by thedrow** [comment]
Is there any documentation yet? This looks really interesting.

**Comment by benoitc** [comment]
@thedrow thanks. I will publish the doc and release a stable version this week-end. Maybe before depending how fast I am in finishing the net api :)
